less/components/footer.less

```diff
@@ -112,8 +112,9 @@ footer a {
 
 	&:hover {
 		text-decoration: none;
-		background-color: @gunmetal;
-		border: 1px solid @gunmetal;
+		background-color: #FFF;
+		border: 1px solid #FFF;
+		color: #000
```

Comment:
missing `;` here  (and lots of places). When it appears at the end of a block, it's not technically wrong. But we prefer to add it because if you have something like this:

```
.btn {
  color: red;
  border-radius: 2px
}
```

And then in six months you add something, it's very easy to forget to add the `;` to the previous column:

```
.btn {
  color: red;
  border-radius: 2px    <--- easy to forget to add semi-colon
  background: blue
}
```
```

Reply:
Same comment as above -- trying to remove all `@black` variables (which takes longer to spell out than simply #000). Promise that I always finish CSS property / values with a semicolon! :p